README.md

```diff
@@ -6,25 +6,36 @@ It is built using the [Cobra](https://github.com/spf13/cobra) library.
 
 ## Installation
 
-ADC can be installed using the `go install` command:
+### macOS/Linux
 
+You can install ADC using the install script by running:
+
+```shell
+curl -sL https://run.api7.ai/adc/install | sh
 ```
-go install github.com/api7/adc@latest
```

Comment:
Why remove it?

Reply:
For users, it is better to directly download the binary instead of installing with `go install`. Now I think maybe we should keep both.

Reply:
Not really. What's the benefit of keeping an installation approach coupled to a tech stack?